GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -43,6 +43,9 @@ typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable ve
 /// The active configuration for this instance of `GIDVerifyAccountDetail`.
 @property(nonatomic, nullable) GIDConfiguration *configuration;
 
+- (instancetype)initWithConfig:(GIDConfiguration *)config
```

Comment:
Seems like this can be `nil` from the `.m` file. Mark as `nullable`.

Reply:
Done.